string/readme.md

```diff
@@ -29,5 +29,7 @@ A string is a class that contains a char array, but automatically manages it for
 * Revere a string using Stack ---->[C++](/Code/C++/reverse_a_string_using_stack.cpp)
 * Reverse Individual Words in String ----> [Java](/Code/Java/revindivstring.java)
 * Maximum occuring character in a string ---->[C++](/Code/C++/max_count.cpp)
```

Comment:
Sort this list

Reply:
sort list as in how?

Reply:
The list should be in alphabetical order, A, b , c,...